mod_app/migrations/0001b_remove_link_models.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 4.2.5 on 2023-12-18 14:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("mod_app", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.DeleteModel(name="Link"),
+        migrations.DeleteModel(name="Drawing"),
+        migrations.DeleteModel(name="Postcard"),
```

Comment:
See my comment about RemoveField having to be before DeleteModel

Reply:
Yes, this has been fixed in the bib PR; I didn't know about the issue until then because that's when I made some changes to the django workflow/action